src/rohan/ArrayAssignment.java

```diff
@@ -0,0 +1,67 @@
+package rohan;
+
+/*write a single program having different methods.
+
+a) To find Min number from given array
+b) To find Max number from given array
+c) To find Avg of all numbers from give array
+d) To display all numbers [0 to length]
+e) To display all numbers [length to 0]
+g) Method to find average of min and max number from array.*/
+
+public class ArrayAssignment {
+
+	int maxOfArray(int[] arr) {
+		int ans = arr[0];
+		for(int i=0;i<arr.length;i++) {
```

Comment:
Video sent on skype for this code, pls go through it

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files